src/main/java/com/github/javabdd/BDD.java

```diff
@@ -586,6 +586,56 @@ public BDD relprod(BDD that, BDDVarSet var) {
      */
     public abstract BDD satOne(BDDVarSet var, boolean pol);
 
+    /**
+     * Computes the BDD that represents the set of successor states from {@code states}, i.e., the ones reachable by
+     * taking one forward step from states in {@code states}, where this BDD is the (partial) transition relation to
+     * consider. It is assumed that old-state and new-state BDD variables are interleaved in this transition relation.
```

Comment:
It indicates interleaving is required, but not that the old var needs to be before the new var, which is also assumed in the implementation.